js/sites.js

```diff
@@ -203,11 +203,42 @@
       let enhanced = this.link;
```

Comment:
I asked @emtwo about this and tried doing some JS in the console:
```javascript
a = {a: 1}
// returns Object { a: 1 }
true && a
// returns Object { a: 1 }
false && a
// returns false
```

So, the original code did evaluate to either `false` or to the result of `getEnhancedLink`.
That said, the semantics of the variable `enhanced` changed. It is now only a shorthand for a link.

It should be removed, and `this.link` should be used instead.